src/Tests/ImageBundleTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\media\Tests\ImageBundleTest.
+ */
+
+namespace Drupal\media\Tests;
+
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+use Drupal\simpletest\WebTestBase;
+
+/**
+ * Ensures that media bundle for images can be created.
+ *
+ * @group media
+ */
+class ImageBundleTest extends WebTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = ['media', 'media_entity', 'media_entity_image', 'image', 'node'];
+
+  /**
+   * The test media bundle.
+   *
+   * @var \Drupal\media_entity\MediaBundleInterface
+   */
+  protected $testBundle;
+  /**
+   * Contains the media bundle storage
+   *
+   * @var \Drupal\media_entity\MediaBundleInterface
+   */
+  protected $bundleStorage;
+
+  /**
+   * The test user.
+   *
+   * @var \Drupal\User\UserInterface
+   */
+  protected $adminUser;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    $entityManager = $this->container->get('entity.manager');
+    $this->bundleStorage = $entityManager->getStorage('media_bundle');
+    $this->testBundle = $this->bundleStorage->load('image');
+
+    $this->adminUser = $this->drupalCreateUser([
+        'view media',
+        'create media',
+        'update media',
+        'update any media',
+        'delete media',
+        'delete any media',
+        'access media overview',
+    ]);
+    $this->drupalLogin($this->adminUser);
+  }
+
+  /**
+   * Tests image media bundle creation from config files.
+   */
+  public function testMediaBundleCreationFromModule() {
+    $type_configuration = array(
+      'source_field' => 'field_image',
+      'gather_exif' => false
+    );
+
+    $this->assertTrue((bool) $this->testBundle, 'The media bundle from default configuration has been created in the database.');
+    $this->assertEqual($this->testBundle->get('label'), 'Image', 'Correct label detected.');
+    $this->assertEqual($this->testBundle->get('description'), 'Use Image for uploading locally hosted images.', 'Correct description detected.');
+    $this->assertEqual($this->testBundle->get('type'), 'image', 'Correct plugin ID detected.');
+    $this->assertEqual($this->testBundle->get('type_configuration'), $type_configuration, 'Type configuration correct.');
+    $this->assertEqual($this->testBundle->get('field_map'), [], 'Correct field map detected.');
+  }
+
+  /**
+   * Tests item creation and thumbnail
+   */
+  public function testMediaBundleItemCreation() {
+    // Define the media item name.
+    $name = $this->randomMachineName();
+    $image_files = $this->drupalGetTestFiles('image');
+    $testImage = File::create((array) current($image_files));
+    $file_path = $this->container->get('file_system')->realpath($testImage->getFileUri());
```

Comment:
You can just do:
$testImage = current($image_files);
$file_path = $this->container->get('file_system')->realpath($testImage->uri);